net-mgmt/zabbix-agent/src/opnsense/mvc/app/models/OPNsense/ZabbixAgent/ZabbixAgent.xml

```diff
@@ -0,0 +1,130 @@
+<model>
+    <mount>//OPNsense/ZabbixAgent</mount>
+    <version>1.0.0</version>
+    <description>
+        Enterprise-class open source distributed monitoring agent
+    </description>
+    <items>
+        <settings>
+            <main>
+                <enabled type="BooleanField">
+                    <default>0</default>
+                    <Required>Y</Required>
+                </enabled>
+                <hostname type="TextField">
+                    <Required>Y</Required>
+                    <mask>/^.{1,255}$/u</mask>
+                    <ValidationMessage>Should be a string between 1 and 255 characters.</ValidationMessage>
+                </hostname>
+                <serverList type="CSVListField">
+                    <Required>Y</Required>
+                    <multiple>Y</multiple>
+                    <mask>/^([a-zA-Z0-9\.:\[\]\s\-]*?,)*([a-zA-Z0-9\.:\[\]\s\-]*)$/</mask>
+                    <ChangeCase>lower</ChangeCase>
+                    <ValidationMessage>Please provide valid Zabbix server addresses, i.e. 10.0.0.1 or zabbix.example.com.</ValidationMessage>
+                </serverList>
+                <listenPort type="IntegerField">
+                    <default>10050</default>
+                    <MinimumValue>1024</MinimumValue>
+                    <MaximumValue>65535</MaximumValue>
+                    <Required>Y</Required>
+                </listenPort>
+                <listenIP type="CSVListField">
+                    <Required>Y</Required>
+                    <multiple>Y</multiple>
+                    <mask>/^((([0-9a-zA-Z._\-]+:[0-9a-zA-Z._\-]+)([,]){0,1}))*/u</mask>
+                    <ChangeCase>lower</ChangeCase>
+                    <ValidationMessage>Please provide valid IP addresses, i.e. 10.0.0.1.</ValidationMessage>
+                </listenIP>
+                <sourceIP type="TextField">
+                    <Required>N</Required>
+                    <mask>/^.{1,255}$/u</mask>
+                    <ValidationMessage>Should be a string between 1 and 255 characters.</ValidationMessage>
+                </sourceIP>
+                <logFileSize type="IntegerField">
+                    <default>100</default>
+                    <MinimumValue>1</MinimumValue>
+                    <MaximumValue>5000</MaximumValue>
+                    <Required>Y</Required>
+                </logFileSize>
+                <debugLevel type="OptionField">
+                    <default>val_3</default>
+                    <OptionValues>
+                        <val_0>0 - basic information</val_0>
+                        <val_1>1 - critical information</val_1>
+                        <val_2>2 - error information</val_2>
+                        <val_3>3 - warnings [default]</val_3>
+                        <val_4>4 - debugging</val_4>
+                        <val_5>5 - extended debugging</val_5>
```

Comment:
Well, in Zabbix the _number_ is actually the debug level. The text is just a vague description. I'm not against removing it, it's just that this is the most precise information...

Regarding the brackets, do you have some examples? I've not seen other GUI elements where the default value was highlighted in any way.